.github/workflows/update-helm-index.rb

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env ruby
```

Comment:
I think this would be useful in `kubecf-tools`. Please move it there.

Reply:
Removed file completely; found [chart-releaser-action](https://github.com/helm/chart-releaser-action) instead, which looks like it does the same thing (using GitHub Releases for storage), is from upstream helm (instead of a random person), and we don't have to maintain it.  If we _do_ want to keep this, we can just grab it from [eirinix-helm-release](https://github.com/cloudfoundry-incubator/eirinix-helm-release/blob/master/bin/update-helm-index.rb) instead.

I should have removed it already anyway because we moved to only building the image here (and not the chart).

Reply:
Removed in 27470adc887896e5c691becb3f9c5eeedffab05b